users/jobs/daily/scrape.py

```diff
@@ -106,7 +106,7 @@ def execute(self,soup,c):
             match = pattern.match(substituted)
             TOTAL = int(match.group(1))
             print("Total: {}".format(TOTAL))
-            conn.close()
+            #conn.close()
```

Comment:
Is this done deliberately? I guess it should be closed.

Reply:
no it shouldn't be closed
if uncommented then it is useful in checking the total number of users without adding them to the database

Reply:
should i remove the commet 
but it would be helpful in future

Reply:
The total number would be fixed for now. So it should be closed i.e., it should be uncommented.

Reply:
magar agar close kar dunga toh fir daily job ka kya fayada
fir daily us script ke run karne pe the databse won't be updated